### PR TITLE
Update readme, add Code of Conduct, Open Collective and license

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @AnXh3L0 @dmichl @cameralibre @evalica :)

(I’d propose changing the license to AGPLv3 from the current Apache btw.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,16 +64,48 @@ jekyll serve --watch --config _config.yml,_config-dev.yml
 
 ### Updating
 
-Updating the various repos can be done with running the this script
+Updating the various repositories can be done with running the this script
```
Comment (cameralibre) [inline README.md]:
```suggestion
Updating the various repositories can be done by running this script
```

File: README.md [hunk 2]
```diff
@@ -64,16 +64,48 @@ jekyll serve --watch --config _config.yml,_config-dev.yml
...
+<a href="https://opencollective.com/opensourcedesign/sponsor/3/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/3/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/4/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/4/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/5/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/5/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/6/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/6/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/7/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/7/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/8/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/8/avatar.svg"></a>
+<a href="https://opencollective.com/opensourcedesign/sponsor/9/website" target="_blank"><img src="https://opencollective.com/opensourcedesign/sponsor/9/avatar.svg"></a>
```
Comment (evalica) [inline README.md]: The sponsor has a weird spacing in my preview

File: README.md [hunk 3]
```diff
@@ -64,16 +64,48 @@ jekyll serve --watch --config _config.yml,_config-dev.yml
...
+
+
+## ♥ Code of Conduct
+
+Please note that Open Source Design has a [Contributor Code of Conduct](https://opensourcedesign.net/code-of-conduct/). By participating in this project online or at events you agree to abide by its terms.
+
+
+## 📜 License
+
+- **🔀 You can use & modify everything as long as you credit [Open Source Design](https://opensourcedesign.net) and use the same license for your resulting work.** [Code license is AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) and content is [Creative Commons Attribution-ShareAlike](https://creativecommons.org/licenses/by-sa/4.0/).
```
Comment (evalica) [inline README.md]: I'm not really sure how easily is to change the code from one license to another; OR the differences between them, but I trust you did an informed decision :P

**Comment by evalica** [comment]
It's nice to have the backers listed. Thanks

File: README.md (hunk 3 above)
Comment (jancborchardt) [reply README.md]: Basically, Apache2 is like Creative Commons Attribution, whereas AGPL is like Creative Commons Attribution-ShareAlike, meaning remixed work has to be licensed under the same license again, having it perpetually in the open. For our website, which is mostly informative lists, this is definitely desirable?

File: README.md (hunk 2 above)
Comment (jancborchardt) [reply README.md]: Don’t think this can be changed as it comes from Open Collective defaults. :\

**Comment by jancborchardt** [comment]
Anyone up for a second review here too? :slightly_smiling_face: @dmichl @eupiteco @aaqaishtyaq @simison

**Pull request merged**